cmd/istioctl/register.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+	"k8s.io/api/core/v1"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"istio.io/pilot/platform/kube"
+)
+
+var (
+	registerCmd = &cobra.Command{
```

Comment:
I think the UX matches the 0.2 use case. this is also why creating the svc is a warning btw - I could add a flag to make it an optional error but I think that just gets in the way of the user for no reason ? (which pr?)